website/content/docs/k8s/installation/install.mdx

```diff
@@ -20,7 +20,7 @@ To learn more about the general architecture of Consul on Kubernetes, scroll
 down to the [architecture](/docs/k8s/installation/install#architecture) section.
 If you would like to get hands-on experience testing Consul as a service mesh
 for Kubernetes, check the guides in the [Getting Started with Consul service
-mesh](https://learn.hashicorp.com/consul/gs-consul-service-mesh/understand-consul-service-mesh?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) track.
+mesh](https://learn.hashicorp.com/tutorials/consul/service-mesh-deploy?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) track.
```

Comment:
Is there a reason we use all the additional sources and parameters? Does it help with tracking?

Reply:
Yeah, exactly. These are the params we want to feed so Heap can correlate our user's journey across properties.